src/dlgprefshoutcast.cpp

```diff
@@ -257,7 +293,35 @@ void DlgPrefShoutcast::slotApply()
     m_pConfig->set(ConfigKey(SHOUTCAST_PREF_KEY, "custom_artist"), ConfigValue(custom_artist->text()));
     m_pConfig->set(ConfigKey(SHOUTCAST_PREF_KEY, "custom_title"),  ConfigValue(custom_title->text()));
     m_pConfig->set(ConfigKey(SHOUTCAST_PREF_KEY, "metadata_format"), ConfigValue(metadata_format->text()));
+}
+
+void DlgPrefShoutcast::setDialogEnabled(bool enabled) {
+        comboBoxServerType->setEnabled(enabled);
```

Comment:
Actually How to keep enabled possibility to shut down shoutcast connection? Overload setenabled first call parent and then enable shoutcast connection enable disable? Any ideas?